kDriveCore/Data/Database/RealmAccessor.swift

```diff
@@ -21,11 +21,20 @@ import Foundation
 import InfomaniakCoreDB
 import RealmSwift
 
+/// Something to identify specific instances of Transactionable
+public enum kDriveDBID {
```

Comment:
I'm open to discuss the naming of this enum

Reply:
Naming feels weird with the 4 caps at the end but I don't have a better idea. If we come up with a better idea we can change it later 👍